oidc/context_processor.py

```diff
@@ -1,15 +1,20 @@
 from django import template
+from django.contrib.auth.middleware import RemoteUserMiddleware
 from django.conf import settings
 
 def has_admin_claim_group(request):
     r_context = {}
     r_context['has_admin_claim_group'] = False
     try:
         c_group = settings.OIDC_DESKTOP_CLAIM_GROUP
-        if c_group in request.session['claim_groups']:
+
+        # This check is in addition to the check done by openresty and acts as
+        # a redundant check for added security
+        groups = request.META.get(settings.GROUPS_META_VAR,'').split('|')
```

Comment:
this can produce a result of `['']`. We should instead fail out with an empty list